rca/utils/models.py

```diff
@@ -719,3 +719,21 @@ def clean(self):
             "Contact",
         )
     ]
+
+
+class LargeCTAMixin(ContactFieldsMixin):
```

Comment:
Suspect use of inheritance, `LargeCTAMixin` is a `ContactFieldsMixin`...? :thinking: 

They seem like different models that serve a similar, but not identical, purpose, even if they have the same field types. Using inheritance + hack ties them together when no relationship necessarily exists, risking future maintenance issues if one were to change. Any reason not to KIS and have separate models?

Reply:
Yea, you're right - there no real need to work the models this way - have re-done this with a model form